.storybook/config.js

```diff
@@ -13,16 +13,12 @@ injectGlobal([], {
   }
 })
 
-const Decorator = props => (
+addDecorator(story => (
   <ThemeProvider>
     <Box p={3}>
-      {props.children}
+      {story()}
```

Comment:
So, there's an issue where when updating a component, storybook's hot loading loses the theme – I was trying to see if this would fix it, but still needs some investigation